app/admin/institution.rb

```diff
@@ -16,11 +16,14 @@
 
   ## Index
   #
-  includes :antennes, :subjects, :advisors, :experts, :sent_matches, :received_matches, :logo
   config.sort_order = 'slug_asc'
 
   controller do
     defaults :finder => :find_by_slug!
+
+    def scoped_collection
+      super.preload :advisors, :subjects, :institutions_subjects, :logo, :antennes, :experts
```

Comment:
c'est le `preload` qui fait une différence alors ? 

Et le `scoped_collection`, ça change quoi ?

Reply:
Dans active admin la plupart du temps on met une ligne 'includes :solicitation, :machinchose...` mais c'est assez lent dans certains cas, le preload est plus rapide d'après les tests que j'ai fait et évite les N+1. et cette methode c'est le moyen de le mettre dans active admin

Reply:
https://stackoverflow.com/questions/8509815/eager-loading-associated-models-in-activeadmin-sql-query

Reply:
Excellent, j'apprends des choses, je kiffe :)

Reply:
par contre ça ne fonctionne pas sur la page admin/territories si on fait un preload de quoique ce soit c'est beaucoup plus lent. On est condamné aux requêtes n+1 sur cette page

Reply:
Tant pis, c'est déjà bien mieux :)